cars.rb

```diff
@@ -0,0 +1,17 @@
+class Cars
```

Comment:
Classes are generally named in a singular way unless the objects they are describing would represent a collection of things. In this case, you instantiate a single car so it should be `Car`.

Reply:
Got it, thank you. Should I correct this before merging?